new_report.html

```diff
@@ -0,0 +1,361 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <title>ReportsWebApp</title>
+
+        <!-- CSS -->
+        <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Roboto:400,100,300,500">
+        <link rel="stylesheet" href="assets/bootstrap/css/bootstrap.min.css">
```

Comment:
you shouldn't add 3rd party files/libraries to repository.
for external css you can use CDNs or bundled into single file packages installed with bower/npm.
For JS libs you can setup webpack (or use another bundler) to bundle npm libraries (such as jquery) to a single file